holoviews/core/dimension.py

```diff
@@ -41,6 +41,25 @@ def param_aliases(d):
     return d
 
 
+def process_dimensions(kdims, vdims):
+    """
+    Processes kdims and vdims specifications into a dictionary
+    of dimensions which can be passed to params.
+    """
+    dimensions = {}
+    for group, dims in [('kdims', kdims), ('vdims', vdims)]:
+        if dims is None:
+            continue
+        elif isinstance(dims, (tuple, basestring, Dimension)):
+            dims = [dims]
+        elif not isinstance(dims, list):
+            raise ValueError("%s must be a Dimension or list of dimensions, "
+                             "specified as tuples, string or Dimension instances, "
+                             "not %s." % (group, dims))
+        dimensions[group] = [d if isinstance(d, Dimension) else Dimension(d) for d in dims]
+    return dimensions
+
```

Comment:
Just checking that this function is confined to ``core``....

Reply:
Good thing to check but I'm fairly certain.